src/custom-element.js

```diff
@@ -307,7 +307,7 @@ function createBaseCustomElementClass(win) {
       return this.upgradeState_ == UpgradeState.UPGRADED;
     }
 
-    /** @return {!Promise} */
+    /** @return {!Promise<!Object>} */
```

Comment:
I think this one will still say `!Promise` based on the `whenSignal` API. You'd have to call `.then(() => x)` to change the response. But I don't see yet that we need to modify this API.